### DISCUSSION TITLE
BOM Light Unit #2 Rev.4 | changed parts

### DISCUSSION THREAD

**Discussion by 1N4003** [srozum/film_camera_tester#36] [Q&A] [answered]
Hi,

I just wanted to ask if the changed parts between Light Unit #2 Rev.3 and Rev.4. (Resistor 3.9K 0.125W & Capacitor 330µF 25V) make a significant difference operation wise. If so, where does the changed capacitor and resistor go? The BOM section for Rev.4 doesn't mention positions and the BOM.csv file is still on Rev.3

Thanks in advance!

**Comment by Surreal62** [comment]
Appears the 330uf cap is placed into C2 on the PCB.
Cannot find where the 3.9K resistor is to be placed as R1 = 2k, R2 = 2.2k and R3, R4 are 4.7K.

**Comment by srozum** [comment, marked as answer]
Was answered in #15

**Reply by 1N4003** [reply]
Oh, ok. Didn’t see that, sorry. Still learning the ropes here. I’m curious, what was the reason for revision #4 of the light source and the partial exchange of parts? Thanks for the awesome project by the way!
